cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControl.java

```diff
@@ -91,11 +91,13 @@ public class CruiseControl extends AbstractModel implements SupportsMetrics, Sup
     protected static final String TLS_CC_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cc-certs/";
     protected static final String TLS_CA_CERTS_VOLUME_NAME = "cluster-ca-certs";
     protected static final String TLS_CA_CERTS_VOLUME_MOUNT = "/etc/cruise-control/cluster-ca-certs/";
-    protected static final String LOG_AND_METRICS_CONFIG_VOLUME_NAME = "cruise-control-metrics-and-logging";
-    protected static final String LOG_AND_METRICS_CONFIG_VOLUME_MOUNT = "/opt/cruise-control/custom-config/";
+    protected static final String CRUISE_CONTROL_CONFIG_VOLUME_NAME = "config";
+    protected static final String CRUISE_CONTROL_SERVER_CONFIG_FILENAME = "server-config";
+    protected static final String CRUISE_CONTROL_CAPACITY_CONFIG_FILENAME = "capacity-config";
```

Comment:
Maybe you should give it some filename-like name then? `server.config`, `capacity.config`?

Reply:
We have log4j2.properties, so I would expect the same file names of the CC repository: capacity.json and cruisecontrol.properties. Same in related tests.

Reply:
Updated based on suggestions